CodePush.m

```diff
@@ -34,52 +34,58 @@ @implementation CodePush {
 
 // These keys are used to inspect/augment the metadata
 // that is associated with an update's package.
+static NSString *const BinaryBundleDateKey = @"binaryDate";
 static NSString *const PackageHashKey = @"packageHash";
 static NSString *const PackageIsPendingKey = @"isPending";
 
+// These values are used to save the bundleURL and extension for the JS bundle
+// in the binary.
+static NSString *bundleResourceExtension = @"jsbundle";
```

Comment:
I know this is unrelated to your PR, but could you create a new pragma section called "Static variables" that includes these two and the three above? I just noticed that I grouped all of these into a "Private constants" section, which isn't accurate.